docs/catalog/cloud/review-platform-and-service-updates-regularly.md

```diff
@@ -0,0 +1,33 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Review platform and service updates regularly
+
+## Description
+Platform updates enable you to use the latest functionality and features to help increase efficiency. Running on outdated software can result in running a suboptimal workload with unnecessary performance issues. New software tends to be more efficient in general.
+
+## Solution
+Upgrade to newer and more efficient services as they become available.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, regularly updating your platforms and services will impact:
+
+- `E`:  Using the latest functionality and features helps increase energy efficiency.
+
+## Assumptions
+Update & Upgrades processed are automated.
+
+## Considerations
+Consider backward compatibility and hardware reusability. An upgrade may not be the most efficient solution if the hardware or the OS isn't supported.
```

Comment:
Even if the hardware and OS are supported, an upgrade doesn't necessarily mean more efficient. Software changes are typically driven from monetary perspective of the developer which may or may not align with the emissions perspective and efficiency for the user.

Reply:
I agree with Bill; new updates may not imply performance improvements and can be purely security/vulnerability fixes, which adds to the overhead of installation/patching all affected compute resources.